conan/tools/google/bazel.py

```diff
@@ -1,44 +1,50 @@
-from conan.tools.files.files import load_toolchain_args
+import os
+import platform
 
+from conan.tools.google import BazelToolchain
 
-class Bazel(object):
-    def __init__(self, conanfile, namespace=None):
-        self._conanfile = conanfile
-        self._namespace = namespace
-        self._get_bazel_project_configuration()
-
-    def configure(self, args=None):
-        pass
-
-    def build(self, args=None, label=None):
-        # TODO: Change the directory where bazel builds the project (by default, /var/tmp/_bazel_<username> )
 
-        bazelrc_path = '--bazelrc={}'.format(self._bazelrc_path) if self._bazelrc_path else ''
-        bazel_config = " ".join(['--config={}'.format(conf) for conf in self._bazel_config])
+class Bazel(object):
 
-        # arch = self._conanfile.settings.get_safe("arch")
-        # cpu = {
-        #     "armv8": "arm64",
-        #     "x86_64": ""
-        # }.get(arch, arch)
-        #
-        # command = 'bazel {} build --sandbox_debug --subcommands=pretty_print --cpu={} {} {}'.format(
-        #     bazelrc_path,
-        #     cpu,
-        #     bazel_config,
-        #     label
-        # )
-        command = 'bazel {} build {} {}'.format(
-            bazelrc_path,
-            bazel_config,
-            label
-        )
+    def __init__(self, conanfile):
+        self._conanfile = conanfile
 
+    def build(self, target="//...", cli_args=None):
```

Comment:
To try to break as less as we can to people that already used the previous implementation maybe we should consider:
- Not changing `label` to `target` as `label` seems to  be aligned with Bazel terminology (https://bazel.build/concepts/labels).
- Not removing the configure method in the new implementation, just outputting a warning saying it's deprecated and that it will be removed in the future but at least it won't crash for users that update
I think it would be good that a package created with the `conan new` template from previous versions would work with the new implementation or at least need minimal changes. What do you think?

Reply:
I know what you mean, but, IMO, the correct term for that is "target" as it shows the help message:
```
$ bazel help build
                                                  [bazel release 6.3.2-homebrew]
Usage: bazel build <options> <targets>

Builds the specified targets, using the options.

See 'bazel help target-syntax' for details and examples on how to
specify targets to build.
```
The most correct term would be `target_label` because it fully represents it and appears on the getting started most of the time.